app/middleware/getGps.js

```diff
@@ -25,6 +26,8 @@ function mapResults(db, res, documents, searchTerm) {
     gp.filterGps = gpDataMapper.mappedTitleForGps(gpDataMapper.getFilteredGps(gp, searchTerm));
     return gp;
   });
+  res.locals.resultsHeader = resultsFormat.pluraliseSurgeryQuestion(res.locals.gps.length);
```

Comment:
I'm surprised the lint doesn't complain about this and the next line about `param-reassign`